Sources/FoundationEssentials/FileManager/FileManager+Files.swift

```diff
@@ -484,7 +484,13 @@ extension _FileManagerImpl {
 #endif
     }
 
-#if !os(Windows)
+#if os(Windows)
+#elseif os(WASI)
```

Comment:
Should we do the same thing that we do on Windows for these functions and prevent calling it at all rather than returning `nil`?

Reply:
Ok, we can do that.